client.go

```diff
@@ -156,7 +157,12 @@ func (c *UDPClient) Run(ctx context.Context) {
 	c.init()
 
 	ticker := utils.NewJitteredTicker(c.FlushInterval)
-	defer ticker.Stop()
+	var background errgroup.Group
+	defer func() { _ = background.Wait() }()
+	background.Go(func() error {
+		ticker.Run(ctx)
+		return nil
+	})
```

Comment:
surely ticker.Stop could do this so we don't have to make every NewJitteredTicker usage do this dance.